src/main/resources/application.yaml

```diff
@@ -8,4 +8,14 @@ logging:
 
 ecatalog:
   products:
-    source-path: src/main/resources/static/products.json
+    sources:
+      - name: Uncle
+        url: https://www.dropbox.com/scl/fi/mhq4vmv42x1hy5k430ae3/products.json?rlkey=00w6n2cfjk8p5wwizz6c4tckr&dl=1
+        format: json
+      # I commented this one because there is invalid annotation of xml elements(tag cannot start with number) so it could not be read.
```

Comment:
You are free to update this link to anything u want. Your team mates fixed the origin file and mapped it to different link, and it worked.

PS
Ofc u can read even the original file. There are bunch of different xml readers for java. But ok, it's borring so variant with updating XML is ok.